vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -528,6 +528,8 @@ class MessageItemFactory @Inject constructor(
         )
 
         val playable = messageContent.mimeType == MimeTypes.Gif
+        // don't show play button because detecting animated webp isn't possible via mimetype
+        val playableIfAutoplay = playable || messageContent.mimeType == MimeTypes.Webp
```

Comment:
This works to autoplay animated WebP in the timeline if autoplay is enabled. If it _isn't_ enabled, no play button shows (because `playable` is `false`) which will prevent confusion with static WebP. Static WebP functions fine in autoplay mode still.